lit_gpt/speed_monitor.py

```diff
@@ -163,6 +164,15 @@ class SpeedMonitorBase:
     | `time/total`                        | Total elapsed time (time/train + time/val)                |
     +-------------------------------------+-----------------------------------------------------------+
 
+    Notes:
+        - The implementation assumes that devices are homogeneous as it normalizes by the world size.
+        - Tokens/sec, flops/sec and MFU do not account for padding tokens if present. We suggest using samples/sec or
+            batches/sec to measure throughput under this circumstance.
```

Comment:
FYI: If you ever wanted to have this rendered properly by sphinx, you'd have to align the text like this for the bullet points. 

```suggestion
        - Tokens/sec, flops/sec and MFU do not account for padding tokens if present. We suggest using samples/sec or
          batches/sec to measure throughput under this circumstance.
```